src/main/java/org/spongepowered/api/entity/hanging/Hanging.java

```diff
@@ -43,7 +43,7 @@
      * Sets the facing direction of this entity.
      *
      * @param direction The direction to face
-     * @param forced Whether physics is ignored
+     * @param forced    Whether physics is ignored
```

Comment:
Yeah, you are right. We just want to make sure nothing slips attention because it is less important.